src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,43 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private final int maxValue = 10;
+    private K[] keyArray = (K[]) new Object[maxValue];
+    private V[] valueArray = (V[]) new Object[maxValue];
+    private int size = 0;
```

Comment:
0 ші a default value
```suggestion
    private int size;
```

Reply:
ok, thank you:)